config/enabled.yml

```diff
@@ -1822,6 +1822,16 @@ Bundler/DuplicatedGem:
     - '**/Gemfile'
     - '**/gems.rb'
 
+Bundler/InsecureProtocolSource:
+  Description: >-
+                 The source `:gemcutter`, `:rubygems` and `:rubyforge` are deprecated
+                 because HTTP requests are insecure. Please change your source to
+                 'https://rubygems.org' if possible, or 'http://rubygems.org' if not.
```

Comment:
How is using `http://rubygems.org` better than using `:rubygems`?

Reply:
Yes. It is preferable to use `source 'http://rubygems.org'` instead of specifying a symbol such as `source :rubygems`.
Of course, the best thing is to use `source 'https://rubygems.org'`.

The following is an example of converting symbol into HTTP strings.

If `source :rubygems` is specified for Gemfile, `remote: http://rubygems.org/` (HTTP) is output to Gemfile.lock.
This is the same when specified the string `source 'http://rubygems.org'`.

```
% cat Gemfile
# frozen_string_literal: true

source :rubygems

gem 'rubocop'

% head Gemfile.lock
GEM
  remote: http://rubygems.org/
  specs:
    ast (2.3.0)
    parallel (1.12.0)
    parser (2.4.0.0)
      ast (~> 2.2)
    powerpack (0.1.1)
    rainbow (2.2.2)
      rake
```

On the other hand, If `source 'https://rubygems.org'` is specified for Gemfile, `remote: https://rubygems.org/` (HTTPS) is output to Gemfile.lock.

```
% cat Gemfile
# frozen_string_literal: true

source 'https://rubygems.org'

gem 'rubocop'

% head Gemfile.lock
GEM
  remote: https://rubygems.org/
  specs:
    ast (2.3.0)
    parallel (1.12.0)
    parser (2.4.0.0)
      ast (~> 2.2)
    powerpack (0.1.1)
    rainbow (2.2.2)
      rake
```

I'm sorry if I misunderstood the intention of question.